lib/ReactViews/Map/Navigation/MeasureTool.jsx

```diff
@@ -22,23 +22,23 @@ const Cartesian3 = require("terriajs-cesium/Source/Core/Cartesian3.js").default;
 const VertexFormat = require("terriajs-cesium/Source/Core/VertexFormat.js")
   .default;
 
-const MeasureTool = createReactClass({
+export const MeasureTool = createReactClass({
   displayName: "MeasureTool",
   mixins: [ObserveModelMixin],
 
   propTypes: {
     terria: PropTypes.object,
-    t: PropTypes.func,
-    i18n: PropTypes.func
+    t: PropTypes.func
   },
 
   getInitialState() {
+    const { t } = this.props;
     return {
       totalDistanceMetres: 0,
       totalAreaMetresSquared: 0,
       userDrawing: new UserDrawing({
         terria: this.props.terria,
-        messageHeader: "Measure Tool",
+        messageHeader: t("measure.Measure-Tool"),
```

Comment:
The casing is breaking the header now:
![image](https://user-images.githubusercontent.com/9134432/69287099-1c969d00-0c49-11ea-8b58-5a8ddde1a677.png)

Reply:
Let's make it:
`"measure-tool" : "Measure Tool"`
(notice the capitalised `Tool` in the value, to mimic the old text that's being replaced)

And then in the component, consumed like `t("measure.measure-tool")`

Reply:
fixed